Sources/_TestingInternals/Discovery.cpp

```diff
@@ -40,19 +53,12 @@ const void *_Nonnull const SWTTestContentSectionBounds[2] = {
 #include <atomic>
```

Comment:
I can probably strip down these includes now, but will leave that for another PR.